### ISSUE TITLE
External libraries in eclipse PYTHONPATH not recognized in pybot run

### ISSUE THREAD

**Issue by aadir7** [nokia/RED#4]
First , I would like to appreciate the team to have provided a long waited awesome plugin for robotframework.
I was waiting for such an integration with eclipse.
Now i have my design and test automation part of same developmen tenvironment.
No to the issue,

I am adding an external library egg file (Pyvimomi.egg) in eclipse PYTHONPATH.
One of my python library file uses this dependency to query to get vm details.
But it is not recognized during my pybot execution.
Still my PYTHONPATH shows only default set in my SHELL environment.

Can you help me find out the cause of this issue.?

**Comment by KrzysztofJozefowicz** [comment]
Hi
Thanks for kind words, we are working hard to provide RED in a state which could be used by most eclipse users. 
Regarding egg files, we have not used them in our daily work, let me clarify you issue.
Does it happen during test run from RED or generally egg file is not visible even when you start pybot from console/command line?
I assume you have installed egg lib by pip or with following instructions: https://pythonhosted.org/setuptools/easy_install.html

**Comment by aadir7** [comment]
With pybot cli, If i provide egg files it in PYTHONATH it detects and runs the tests.
But with RED + eclipse , Even if i have them in PYTHONPATH, It doesn't detect the classes in the egg files.
and it doesn't detect if i include it as External library in RED project.

**Comment by adam-wypych** [comment]
Hi,

I tried to made test like You on my laptop with Windows 7 64-bit, Python 2.7 64-bit and Robot Framework 3.0 alpha 1, RED 0.4.1, Eclipse Java SDK Mars R1, Java 1.7 JDK update 79. Base on Your description I made steps like below:
1. Add in System Variables `PYTHONPATH` variable with value: `C:\Python27\;C:\Python27\Lib;C:\Python27\Scripts;D:\userdata\wypych\Desktop\Pyvmomi_test\pyvmomi-5.5.0.2014.1.1-py2.7.egg`
2. I have been started RED and create new project.
3. I have been created wrapper library, which only tries to import something from pyvmomi - filename `PyvmomiLibraryWrapper.py` inside project:
```python
class PyvmomiLibraryWrapper(object):
	def setup(self):
		from pyVmomi import Version
```
4. I have created file `LibTest.robot` with content like below:
```robot
*** Settings ***
Library           PyvmomiLibraryWrapper.py

*** Test Cases ***
TestLib
    Setup
```
5. I used `Quick Fix` to do this I just put keyboard cursor to be inside  `PyvmomiLibraryWrapper.py` and I press on my keyboard (`Ctrl` + `1`)  to import line to `libspec` for `PyvmomiLibraryWrapper.py` and recognize my own library keywords and error prompt to disappear.
